channels/fo.m3u

```diff
@@ -1,10 +1,6 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="KVF.fo" tvg-name="KVF" tvg-country="FO" tvg-language="" tvg-logo="" group-title="",KVF (720p)
 https://w1.kringvarp.fo/uttanlands/_definst_/smil:uttanlands.smil/chunklist.m3u8
```

Comment:
Maybe keep this one : https://w1.kringvarp.fo/uttanlands/smil:uttanlands.smil/playlist.m3u8

Contains 4 different resolutions.